test/integration_test/main_test.go

```diff
@@ -170,3 +252,125 @@ func setup() error {
 
 	return nil
 }
+
+func initAetosDashboard(enableDash bool, user string, testDescription string, testType string,
+	testTags string, testBranch string, testProduct string, testsetID int) error {
+	logrus.Infof("Initializing Aetos Dashboard!")
+	dash = aetosutil.Get()
+	if enableDash && !isDashboardReachable() {
+		enableDash = false
+		logrus.Infof("Aetos Dashboard is not reachable. Disabling dashboard reporting.")
+	}
+
+	dash.IsEnabled = enableDash
+	testSet := aetosutil.TestSet{
+		User:        user,
+		Product:     testProduct,
+		Description: testDescription,
+		Branch:      testBranch,
+		TestType:    testType,
+		Tags:        make(map[string]string),
```

Comment:
create a tags map for string : string. Pass operator-test:"true" for now.

Reply:
That is already part of TestCaseBegin call. In case it's user provided, we have handled the same in deploy.sh